Buildings/Fluid/FixedResistances/BuriedPipes/Examples/PipeGroundCoupling.mo

```diff
@@ -0,0 +1,155 @@
+within Buildings.Fluid.FixedResistances.BuriedPipes.Examples;
+model PipeGroundCoupling "Example model for pipe ground coupling model"
+  extends Modelica.Icons.Example;
+  replaceable package Medium = Buildings.Media.Water "Medium in the pipe" annotation (
+      choicesAllMatching=true);
+  parameter Integer nSoi = 4 "Number of probed depths";
+  parameter Modelica.Units.SI.Length dep[nSoi]={0,2,5,9} "Probed depths";
+  parameter Modelica.Units.SI.Length length=100 "Pipe length";
+  parameter Modelica.Units.SI.MassFlowRate m_flow_nominal=3
+    "Mass flow rate";
+  parameter Modelica.Units.SI.Temperature Tin=273.15+11.5 "Mean inlet temperature";
+
+  Buildings.Fluid.FixedResistances.BuriedPipes.PipeGroundCoupling pipeGroundCoupling(
+    lPip=length,
+    rPip=0.1,
+    thiGroLay=1.1,
+    nSta=5,
+    nSeg=1,
+    TpipSta=Tin,
+    TGrouBouSta=Tin,
+    cliCon=cliCon,
+    soiDat=soiDat)
+    annotation (Placement(transformation(extent={{10,60},{32,80}})));
+  Fluid.Sources.MassFlowSource_T           sou(
+    redeclare package Medium = Medium,
+    use_T_in=true,
+    m_flow=m_flow_nominal,
+    T=293.15,
+    nPorts=1) "Flow source"
+    annotation (Placement(transformation(extent={{-62,30},{-42,50}})));
+  Fluid.Sensors.TemperatureTwoPort           senTemIn(
+    redeclare package Medium = Medium,
+    m_flow_nominal=m_flow_nominal,
+    tau=0,
+    T_start=Tin) "Temperature sensor"
+    annotation (Placement(transformation(extent={{-30,30},{-10,50}})));
```

Comment:
Correct formating

Reply:
I formatted all the updated code trying to make it more readable.